[fixed] some untabbable elements are being returned from findTabbable

flurmbo opened the pull request:
Fixes #732 

Changes proposed:

The bug in the example stems from the `hidesContents` function in `tabbable.js` returning false on the button with the `display: none` inline style. It is returning false because the element has the computed style `overflow: visible` (which is default value). 

The problem with this is that having `overflow: visible` doesn't mean there is any content that is  overflowing. 

I think there are a couple possible ways to fix this. One would be to add an additional check for the `display: none` style if `zeroSize === true`, which would directly solve the bug in the example.

I'm not sure that will catch every possibility, so instead I check the elements `scrollWidth` and `scrollHeight` to determine whether or not there is any overflowing content.

Acceptance Checklist:
- [x] The commit message follows the guidelines in `CONTRIBUTING.md`.
- [ ] If this is a code change, a spec testing the functionality has been added.

diasbruno commented:
@flurmbo This is going to be a hard one, but can you try to add a test case for this?

flurmbo commented:
Hi @diasbruno, sure, should the test just test the tabbable helper function or should it recreate the modal in the example and test that focus stays within the modal?

diasbruno commented:
A test case for tabbable would be great.

flurmbo commented:
Hi @diasbruno, I added a test case for the tabbable helper. It also looks like the previous test was editing the wrong DOM elements style (`elem` instead of `input`) so I fixed that in a separate commit.

diasbruno commented:
Thanks, @flurmbo. I'll merge this and make a release tomorrow.
